addons/website_multiple/__openerp__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': "Website Multiple",
+    'category': 'Website',
+    'summary': 'Allow to use multiple website feature.',
```

Comment:
@bst-odoo I think it should be 'Multiple Websites' as using keyword multiple it means plural.